ivy_tests/test_ivy/helpers.py

```diff
@@ -53,6 +62,41 @@
 import ivy.functional.backends.numpy as ivy_np
 
 
+def convtrue(argument):
+    """Convert NativeClass in argument to true framework counter part"""
+    if isinstance(argument, NativeClass):
+        return argument._native_class
+    return argument
+
+
+def convjax(argument):
```

Comment:
...but these and all functions below should be moved into the framework-specific `__init__.py` files